build/system.gyp

```diff
@@ -11,6 +11,19 @@
   },
   'targets' : [
     {
+      'target_name': 'linux_notification',
```

Comment:
Real name of library is 'libnotify'. Never mind, it'll be changed to 'linux_libnotify'.